File: SpringBootApp/src/main/java/com/example/springbootapp/Data/Entities/Embdeddables/OrderInformations.java
package com.example.springbootapp.Data.Entities.Embdeddables;

import jakarta.persistence.Embeddable;
import lombok.Data;

@Data
@Embeddable
public class OrderInformations {
    private String street;
    private String city;
    private String state;
    private String zip_code;
    private String civic_number;
    private String customer_email;
    private String customer_firstName;
    private String customer_lastName;
}

File: ClientAdmin/app/src/main/java/com/example/clientadmin/model/embdeddables/OrderInformations.kt
package com.example.clientadmin.model.embdeddables

data class OrderInformations (
    val street: String,
    val city: String,
    val state: String,
    val zipCode: String,
    val civicNumber: String,
    val customerEmail: String,
    val customerFirstName: String,
    val customerLastName: String
)